### PR TITLE
Temporarily fix Firefox platform problem

### PR DESCRIPTION
**Pull request by ylavoie**
This fixes issue #375 until Firefox/Selenium can negociate again

### CODE REVIEW

**Comment by teodesian** [comment]
The code looks like it should work, but there's issues with the unit tests.  I'll look at that when I can if you don't get to it first.

**Comment by ylavoie** [comment]
They are version dependent. I don't have any problems with `t/01-driver.t` on Artful, Travis runs Trusty.
But on Artful, `t/error.t` goes wild on an empty url.

File: lib/Selenium/Remote/Driver.pm [hunk 1]
```diff
@@ -866,13 +866,14 @@ sub new_session {
...
             'browserName'        => $self->browser_name,
-            'platform'           => $self->platform,
             'javascriptEnabled'  => $self->javascript,
             'version'            => $self->version,
             'acceptSslCerts'     => $self->accept_ssl_certs,
             %$extra_capabilities,
         },
     };
+    $args->{'desiredCapabilities'}{'platform'} = $self->platform
+        if $self->browser_name ne 'firefox' || !$self->{is_wd3};
```
Comment (teodesian) [inline lib/Selenium/Remote/Driver.pm]: what if we actually want to pass a platform other than 'ANY'? does that also cause the geckodriver to die?

Comment (ylavoie) [reply lib/Selenium/Remote/Driver.pm]: I tried imposing a few, same results.

Comment (teodesian) [reply lib/Selenium/Remote/Driver.pm]: Dang.  Sounds like a proper bug in the driver then.

**Pull request merged**